suite-native/accounts/src/components/AccountsList/AccountsListItem.tsx

```diff
@@ -81,8 +87,9 @@ export const AccountsListItem = ({
     }, [account, accountHasAnyTokens, onPress]);
 
     const doesCoinSupportTokens = isCoinWithTokens(account.symbol);
-    const shouldShowAccountLabel = !doesCoinSupportTokens || hideTokens;
-    const shouldShowTokenBadge = accountHasAnyTokens && hideTokens;
+    const shouldShowAccountLabel = !doesCoinSupportTokens || hideBadges;
+    const shouldShowTokenBadge = accountHasAnyTokens && hideBadges;
```

Comment:
The naming is confusing. I would expect it to be 
```suggestion
    const shouldShowTokenBadge = accountHasAnyTokens && !hideBadges;
```

Reply:
Yea, it was confusing I renamed it and it should be more clear now